grafana/telegraf/telegraf.conf

```diff
@@ -0,0 +1,14 @@
+[global_tags]
+
+[agent]
+  interval = "60s"
+  round_interval = true
+  metric_batch_size = 1000
+  metric_buffer_limit = 10000
+  collection_jitter = "0s"
+  flush_interval = "10s"
+  flush_jitter = "0s"
+  precision = ""
+  hostname = "192.168.0.2"
```

Comment:
Does this need to get edited by the used to change the IP address? You README doesn't say. Also, could it be controlled by an environment variable I set in the docker compose file?

Reply:
Oh urgh, good spot - I wonder what that is used for - I forgot about that one...